lib/LoginStrategy.ts

```diff
@@ -32,6 +32,9 @@ import {
   VerificationResult,
 } from '@energyweb/vc-verification';
 import { StatusListEntryVerification } from '@ew-did-registry/revocation';
+import { SiweMessage, SiweResponse } from 'siwe';
+import type { SiweMessage as SiweMessagePayload } from 'siwe';
+import { hasIn } from 'lodash';
```

Comment:
I think `lodash` isn't in the dependencies listed in `package.json`, so this is being accessed without being listed, which should be avoided.
From what I can tell `hasIn`, is most useful if there is a path to traverse which isn't the case at the moment, so I think we can just just straight object indexing